### ISSUE TITLE
Add -slice

### ISSUE THREAD

**Issue by simonepri** [mbloch/mapshaper#239]
## Command description
Given a Polygon or MultiPolygon and a Line divide the polygon in 2 half (if the line passes through the polygon)

**Comment by simonepri** [comment]
What about the properties of the Feature? If we convert it to a line we lose them.

**Comment by nvkelso** [comment]
See also:

-
https://github.com/tilezen/vector-datasource/blob/master/data/tile-shapefile.py

**Comment by meetirfan** [comment]
yes you are right, in above workflow properties of the Features will be gone. 

In my case I am using then QGIS to attached the properties by using spatial join.

**Comment by mbloch** [comment]
I approve of the polygon + polyline `-slice` command. I'd also like to add a polygon + polygon modality to `-slice`, which would slice two polygon layers into a mosaic of smaller polygons and assign properties from both layers to the output shapes.
